### PR TITLE
fix: using ee/ci/base amd64 image

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

File: pipelines/PingCAP-QE/tidb-test/latest/pod-ghpr_integration_typeorm_test.yaml [hunk 1]
```diff
@@ -15,12 +15,9 @@ spec:
           memory: 16Gi
           cpu: "6"
     - name: nodejs
-      image: "hub.pingcap.net/ee/ci/base:latest"
+      image: "hub.pingcap.net/ee/ci/base@v20230810-go1.21.0"
```
Comment (wuhuizuo) [inline pipelines/PingCAP-QE/tidb-test/latest/pod-ghpr_integration_typeorm_test.yaml:18]:
```suggestion
      image: "hub.pingcap.net/ee/ci/base:v20230810-go1.21.0"
```

Comment (wuhuizuo) [reply pipelines/PingCAP-QE/tidb-test/latest/pod-ghpr_integration_typeorm_test.yaml:18]: We can just use one container. the image include go and nodejs stack

Comment (wuhuizuo) [reply pipelines/PingCAP-QE/tidb-test/latest/pod-ghpr_integration_typeorm_test.yaml:18]: and update 
```yaml
spec:
  securityContext:
    fsGroup: 1000
```

to 
```yaml
spec:
  securityContext:
    runAsUser: 0
```

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**